### ISSUE TITLE
log file tensorboard error

### ISSUE THREAD

**Issue by ghost** [CYang0515/pytorch_ssn#1]
Hi, I wanted to see how the loss changed during the course of training, so I tried to load the event files the "logs" directory using tensorboard (tb-nightly version, not Tensorboard X). However, i keep getting the following error message:

" ... core_plugin.py:215] Unable to get first event timestamp for run .: No event timestamp could be found."

So i was wondering whether the log event files in the "logs" directory of this repo are files that actually have been logged?

Thanks in advance :))
